routesrv/polling.go

```diff
@@ -127,3 +126,22 @@ func (p *poller) poll(wg *sync.WaitGroup) {
 		}
 	}
 }
+
+func (p *poller) process(routes []*eskip.Route) []*eskip.Route {
+
+	if p.defaultFilters != nil {
+		routes = p.defaultFilters.Do(routes)
+	}
+	if p.oauth2Config != nil {
+		routes = p.oauth2Config.NewGrantPreprocessor().Do(routes)
+	}
+	for _, editor := range p.editRoute {
+		routes = editor.Do(routes)
+	}
+
+	for _, cloner := range p.cloneRoute {
+		routes = cloner.Do(routes)
+	}
+
+	return routes
```

Comment:
lets also move https://github.com/zalando/skipper/blob/566fe8c4ad7f7d46da7a2bb9b8a1f6bb75125457/routesrv/polling.go#L102-L105 here (no need to check length, sort would be no-op for empty slice)